### PR TITLE
Flight detection

### PR DESCRIPTION
**Pull request by luca7084**
Attempt at using an event to detect the rocket take-off. In the cases where there is only one time driver (such as with the FMU command sequences), it works perfectly. However, when multiple time drivers are added (the case of the CoSApp command sequences), it is still necessary to specify in the initial conditions whether the rocket is flying or not.

### CODE REVIEW

File: rocket_twin/drivers/mission.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def __init__(
         # Init and stop conditions
         init_fuel = init
         init_flight = {
-            "rocket.flying": True,
+            "rocket.flying": 1.0,
```
Comment (GuyDSP) [inline rocket_twin/drivers/mission.py:46]: why did you change it ?

File: rocket_twin/systems/rocket/rocket.py [hunk 2]
```diff
@@ -39,7 +39,14 @@ def setup(self):
         )
         self.connect(self.tank.outwards, self.dyn.inwards, {"weight": "weight_tank", "cg": "tank"})
 
-        self.add_inward("flying", False, desc="Whether the rocket is flying or not", unit="")
+        self.add_inward_modevar("flying", 0.0, desc="Whether the rocket is flying or not", unit="")
```
Comment (GuyDSP) [inline rocket_twin/systems/rocket/rocket.py:42]: why not a boolean ?

**Review by GuyDSP — CHANGES_REQUESTED** [review]

File: rocket_twin/systems/rocket/rocket.py (hunk 2 above)
Comment (luca7084) [reply rocket_twin/systems/rocket/rocket.py:42]: Same as above

Comment (luca7084) [reply rocket_twin/systems/rocket/rocket.py]: Check
